plugins/plugin-site/src/components/PluginHealthScore.jsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
+import React, {useState} from 'react';
 import PropTypes from 'prop-types';
-import {Progress} from 'reactstrap';
+import {Progress, Tooltip} from 'reactstrap';
 
-function PluginHealthScore({healthScore}) {
+function PluginHealthScore({healthScore, name}) {
     const score = healthScore.value || 0;
     const color =
     score > 80 ? 'success' : score > 60 ? 'warning' : 'danger';
+
+    const tooltipId = `tooltip-${name}`;
+    const [tooltipOpen, setTooltipOpen] = useState(false);
+    const toggle = () => setTooltipOpen(!tooltipOpen);
     return (
         <>
             <div>
-                Health Score
+                <div>
+                    Health Score
+                    <a
+                        href={`https://plugin-health.jenkins.io/scores/${name}`}
+                        id={tooltipId}
+                        style={{marginLeft: '5px'}}
+                    >
+                        ?
+                    </a>
+                    <Tooltip
+                        placement="top"
+                        isOpen={tooltipOpen}
+                        autohide={false}
+                        target={tooltipId}
+                        toggle={toggle}
+                    >
+                        This is the health score of the plugin. More information can be found
+                        {' '}
+                        <a
+                            href={`https://plugin-health.jenkins.io/scores/${name}`}
+                        >
+                            here
```

Comment:
```suggestion
                        View the plugin's
                        {' '}
                        <a
                            href={`https://plugin-health.jenkins.io/scores/${name}`}
                        >
                            total health score.
```
Using words like 'here' for URLs isn't a good practice, see https://uxmovement.com/content/why-your-links-should-never-say-click-here/, nor do we need to state that this is the health score, given it does already say so right below the the box.

I recommend moving this to a dedicated element because clicking on the question mark links through the plugin page, rather than directly to the health score page, causing unneeded browser noise.

Reply:
Thanks for informing me about this good practice, and I think I should remove the link from the question mark because I already added a tooltip